src/qml/controls/NodeAttributeList.qml

```diff
@@ -76,11 +75,11 @@ Rectangle {
 
                 width: root.maxDelegateWidth()
                 implicitHeight: childrenRect.height
-                color: ((listViewDelegate.index % 1) == 0) ? root.theme.color1 : root.theme.color2
+                color: root.palette.base
```

Comment:
Do you not want `base`/`alternateBase`?

Reply:
This is not necessary for the NodeAttributeList. Modulu 1 is always 0 :D